### PR TITLE
Support XFS

### PR DESCRIPTION
**Pull request by snormore**
Include `xfsprogs` in the base Docker image so that XFS can be used.

Also validated in https://github.com/snormore/doks-examples/tree/master/xfs

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,7 +34,7 @@ bump-version:
 .PHONY: compile
 compile:
 	@echo "==> Building the project"
-	@env CGO_ENABLED=0 GOOS=${OS} GOARCH=amd64 go build -o cmd/do-csi-plugin/${NAME} -ldflags "$(LDFLAGS)" ${PKG} 
+	@docker run --rm -it -e GOOS=${OS} -e GOARCH=amd64 -v ${PWD}/:/app -w /app golang:1.12-alpine sh -c 'apk add git && go build -o cmd/do-csi-plugin/${NAME} -ldflags "$(LDFLAGS)" ${PKG}'
```
Comment (fatih) [inline Makefile]: I like this, but I think this should be in a separate PR.

Comment (snormore) [reply Makefile]: Good call 👉 https://github.com/digitalocean/csi-digitalocean/pull/162

**Comment by fatih** [comment]
LGTM
